system-int-test/src/cmr/system_int_test/utils/generic_util.clj

```diff
@@ -14,6 +14,16 @@
                    (slurp)
                    (json/parse-string true)))
 
+(def data-quality-summary (-> "schemas/dataqualitysummary/v1.0.0/metadata.json"
+                              (jio/resource)
+                              (slurp)
```

Comment:
Whitespace looks off here

Reply:
In the PR it looks off, but in my editor and when I try to edit it in github directly, the spacing is correct.

Reply:
it was off, I corrected it for erich.